### PR TITLE
[Kotlin Migration] TagsTest.java

### PR DESCRIPTION
**Pull request by Prince-kushwaha**
## Purpose / Description
Kotlin migration of TagsTest.java

- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [x] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/test/java/com/ichi2/libanki/TagsTest.kt [hunk 1]
```diff
@@ -13,71 +13,57 @@
...
+        val tags_list1 = ArrayList<String>()
+        tags_list1.add("Todo")
+        tags_list1.add("todo")
+        tags_list1.add("Needs revision")
+        val tags_list2 = ArrayList<String>()
+        tags_list2.add("Todo")
+        tags_list2.add("todo")
+        tags_list2.add("Needs")
+        tags_list2.add("Revision")
+        Assert.assertNotEquals(tags_list1, tags.split("Todo todo Needs Revision"))
```
Comment (david-allison) [inline AnkiDroid/src/test/java/com/ichi2/libanki/TagsTest.kt:40]: import `assert`

File: AnkiDroid/src/test/java/com/ichi2/libanki/TagsTest.kt [hunk 2]
```diff
@@ -13,71 +13,57 @@
...
+        TestCase.assertEquals(tags_list2, tags.split("Todo todo Needs Revision"))
```
Comment (david-allison) [inline AnkiDroid/src/test/java/com/ichi2/libanki/TagsTest.kt:41]: import `TestCase`

**Review by david-allison — CHANGES_REQUESTED** [review]
Looks good. Two comments

Could you fix the spacing of the code to match how it previously looked?

**Review by david-allison — CHANGES_REQUESTED** [review]
Still line spacing changes which should be reverted to improve clarity

Screenshot 2022-04-16 at 20 45 03

**Review by david-allison — APPROVED** [review]
Thanks!

**Review by mikehardy — APPROVED** [review]
Nice! Thanks

**Pull request merged**